src/antsibull/data/ansible-setup_py.j2

```diff
@@ -171,6 +173,20 @@ setup(
     license='GPLv3+',
     python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
     packages=['ansible_collections'],
+{% if version.major >= 6 %}
+    exclude_package_data={
+        'ansible_collections': [
+{%   for name, namespace in collections %}
+            '{{ name }}/{{ namespace }}/.gitignore',
+            '{{ name }}/{{ namespace }}/.github/*',
```

Comment:
```suggestion
            '{{ name }}/{{ namespace }}/.github/*',
```
Does this mean excluding the whole directory or only its content? If only its content, should we not include empty dirs?

Reply:
It seems to do both. (And yes, it looks strange :) )

Reply:
ok, thanks for clarifying:)